### ISSUE TITLE
Twillo: Missing alt text for twillo logo

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#323]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/twilioortc/

```
<p class="caption powered-by">
										<span>powered by</span>
										<a href="https://www.twilio.com/webrtc" target="_parent"><img src="/img/wordmark.svg"></a>
									</p>

```
Reads powered by wordmark.svg

Fix:

Add alt="Twillo"

(or change to an background image on anchor for a more involved fix)

**Comment by alrra** [comment]
@dstorey @sarvaje This demo is external, namely, it's from https://twilio-ahoy.herokuapp.com/edge-ortc.
